app/src/components/Header.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { useAuth0 } from "@auth0/auth0-react";
+
+import "../styles.css";
+
+export default function Header() {
+  const { logout, user, isAuthenticated } = useAuth0();
+
+  return (
+    <div className="header">
+      <h1>MyGarden</h1>
+      {isAuthenticated ? (
+        <>
+          <main>
+            <nav></nav>
+          </main>
```

Comment:
`<main>` indicates the main body of the page like your Home or User Profile pages. Lines 15-17 should be taken out.